stories/tie.fmf

```diff
@@ -0,0 +1,26 @@
+/new-test:
+    story:
+        As a user I want to link a newly created test to a Jira issue.
+
+    description:
+        User should be able to link a newly created test/plan to JIRA issue.
+        The url for service will be generated by default when linking new test to JIRA issue.
```

Comment:
```suggestion
        The url for service will be generated by default when linking new test to JIRA issue,
        if user has the tmt web service url defined in their `tmt` config.
```

Reply:
What means the name of this file ('tie')?

From this story I'd assume 'button' to add the link to jira via tmt web service - is that correct? Or will it be `tmt tests export` responsibility? 

What is the definition of 'new' test? IMO the '/existing-test' and '/new-test' should be merged into single UC.

Reply:
> What is the definition of 'new' test? IMO the '/existing-test' and '/new-test' should be merged into single UC.

In order to simplify test creation, I think `tmt test create` could create the link directly when creating test files. No extra command would be needed.